src/main/scala/api/MyMap.scala

```diff
@@ -0,0 +1,88 @@
+package api
+
+
+import scala.collection.JavaConverters._
+
+import java.util.concurrent.ConcurrentHashMap
+
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+
+import scala.collection.concurrent
+
+class MyMap {
+  val map: concurrent.Map[String, String] = new ConcurrentHashMap[String, String].asScala
+
+
+  def createResponse(request: HttpServletRequest, response: HttpServletResponse, route: String, returnVal: Object): Unit = {
```

Comment:
`returnVal` should be a String.
Also, this line exceeds 100 characters https://github.com/Sanyaku/codelabs/blob/master/backend/pages/Code-Style.md#line-wrapping